src/main/resources/reference.conf

```diff
@@ -511,6 +511,41 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  spend-profile = {
+    actionPatternObjects = {
+      "create" = {
+        description = "Can create profiles"
+      }
+      "update" = {
+        description = "Can update profiles"
+      }
+      "delete" = {
+        description = "Can delete profiles"
+      }
+      "list" = {
+        description = "Can read profiles"
+      }
+      "link" = {
+        description = "Can link profiles"
+      }
+      "share_policy::owner" = {
+        description = "Can grant and revoke a users' owner permission"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      admin = {
+        roleActions = ["share_policy::owner"]
+      }
+      owner = {
```

Comment:
probably want `share_policy::user` in here unless there are only owners

Reply:
Does this mean we also need to assign the owner & admin a "share_policy::admin" action?

Reply:
yes, but why do we need the admin role?